src/equations/compressible_euler_1d.jl

```diff
@@ -660,6 +660,26 @@ end
     return λ_min, λ_max
 end
 
+"""
+    min_max_speed(u_ll, u_rr, orientation::Integer, equations::CompressibleEulerEquations1D)
+
+Implements the classic 2-wave HLL solver, see the [original paper](https://epubs.siam.org/doi/abs/10.1137/1025002)
+or this [lecture notes, Eq. (9.27)](https://metaphor.ethz.ch/x/2019/hs/401-4671-00L/literature/mishra_hyperbolic_pdes.pdf).
+"""
+@inline function min_max_speed(u_ll, u_rr, orientation::Integer,
+                               equations::CompressibleEulerEquations1D)
+    rho_ll, v1_ll, p_ll = cons2prim(u_ll, equations)
+    rho_rr, v1_rr, p_rr = cons2prim(u_rr, equations)
+
+    c_ll = sqrt(equations.gamma * p_ll / rho_ll)
+    c_rr = sqrt(equations.gamma * p_rr / rho_rr)
+
+    λ_min = min(v1_ll - c_ll, v1_rr - c_rr)
+    λ_max = max(v1_ll + c_ll, v1_rr + c_rr)
+
+    return λ_min, λ_max
+end
```

Comment:
- I'm not sure whether this is a good name. What I remember from this paper is that they need guaranteed upper/lower bounds of all wave speeds. That's not necessarily the case with this estimate, is it?
  What about `min_max_speed_pointwise` or something like that to clarify that it takes the min/max of the pointwise local wave speeds?
- Could you please use the citation style including a DOI that we use i other docstrings, too?
- These comments hold of course for all variants of this.

Reply:
1) True, in the paper by HLL they write (page 50, i)): 
"_Denote by aL and aR lower and upper bounds, respectively, for the smallest and largest signal velocity, calculated according to some algorithm."_

More precisely, according to this [(reletively recent) paper by Toro et. al.](https://doi.org/10.1016/j.compfluid.2020.104640) [[Arxiv Version]](https://arxiv.org/pdf/2005.04898.pdf) the estimates that are currently implemented are credited to [Davis](https://doi.org/10.1137/0909030), so one could call that estimate `min_max_speed_davis` (which would also be a bit more consistent with `min_max_speed_einfeldt`.).

Reply:
Sure, `min_max_speed_davis` sounds good to me.